### PR TITLE
Model updating br

### PR DESCRIPTION
**Pull request by AlexKazmirchuk**
added new model classes, updated some classes, added other services and repositories

### CODE REVIEW

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationEntity.java [hunk 1]
```diff
@@ -14,9 +15,72 @@
...
 
-    //todo add other fields later
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "logo_image_uri")
+    private String logoImageUri;
+
+    @Column(name = "type")
+    private Integer type;
```
Comment (Dominys) [inline softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationEntity.java]: @AlexKazmirchuk Make enum.

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationSettingsEntity.java [hunk 2]
```diff
@@ -15,9 +16,59 @@
 
     private static final long serialVersionUID = 7214265511617637837L;
 
-    //todo add other fields later
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "admin_id")
+    private ProfileEntity admin;
```
Comment (Dominys) [inline softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationSettingsEntity.java]: @AlexKazmirchuk This must be mapped as one-to-one or field must be string.

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/DeviceEntity.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ */
+
+@Entity
+@Table(name = "devices")
+public class DeviceEntity extends BaseEntity implements Serializable {
+
+    private static final long serialVersionUID = -506229924089552354L;
+
+    @Column(name = "last_confirmation_data")
+    private Long lastConfirmationData;
```
Comment (Dominys) [inline softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/DeviceEntity.java]: @AlexKazmirchuk Date not data ))

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/MessageEntity.java [hunk 4]
```diff
@@ -14,9 +15,126 @@
...
 
-    //todo add other fields later
+    @Column(name = "sender_id")
+    private String senderID; // or ProfileEntity sender?
+
+    @Column(name = "conversation_id")
+    private String conversationID; // or Conversation conversation?
+
+    @Column(name = "message_type")
+    private Integer messageType;
```
Comment (Dominys) [inline softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/MessageEntity.java]: @AlexKazmirchuk make enum

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/MessageEntity.java [hunk 5]
```diff
@@ -14,9 +15,126 @@
...
+
+    @Column(name = "payload")
+    private String payload;
+
+    @Column(name = "status")
+    private Integer status;
```
Comment (Dominys) [inline softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/MessageEntity.java]: @AlexKazmirchuk make enum

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ProfileStatusEntity.java [hunk 6]
```diff
@@ -15,9 +16,46 @@
 
     private static final long serialVersionUID = -5692872742644522060L;
 
-    //todo add other fields later
+    @Column(name = "is_online")
+    private Boolean isOnline;
```
Comment (Dominys) [inline softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ProfileStatusEntity.java]: @AlexKazmirchuk We cannot get this information from database.

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationEntity.java (hunk 1 above)
Comment (AlexKazmirchuk) [reply softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationEntity.java]: @Dominys Done! ConversationType enum created

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationSettingsEntity.java (hunk 2 above)
Comment (AlexKazmirchuk) [reply softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationSettingsEntity.java]: @Dominys done. changed to String type

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/DeviceEntity.java (hunk 3 above)
Comment (AlexKazmirchuk) [reply softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/DeviceEntity.java]: @Dominys fixed!)

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/MessageEntity.java (hunk 4 above)
Comment (AlexKazmirchuk) [reply softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/MessageEntity.java]: @Dominys Done! MessageType enum created

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/MessageEntity.java (hunk 5 above)
Comment (AlexKazmirchuk) [reply softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/MessageEntity.java]: @Dominys Done! MessageStatus enum created

File: softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ProfileStatusEntity.java (hunk 6 above)
Comment (AlexKazmirchuk) [reply softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ProfileStatusEntity.java]: @Dominys fixed(removed this field)

**Pull request closed without merging**